### PR TITLE
fix: #1115 Day Reports: A word ‘tag’ is presented instead of ‘label’ …

### PR DESCRIPTION
**Pull request by timoxa20**
…in some cases

Fix changed the tag to label in buttons, dialog box, and tooltips. Added a comma from task #1095

### CODE REVIEW

File: mw-webapp/src/dictionary/WayPageContent.json [hunk 1]
```diff
@@ -498,27 +498,27 @@
...
+      "ka": "ამოღება ამოცანა ეტიკეტები"
     },
     "deleteJobTagQuestion": {
-      "ru": "Вы уверены, что хотите удалить $jobTag из тэгов?",
-      "en": "Are you sure you want to remove $jobTag from job tags?",
-      "ua": "Ви впевнені, що хочете видалити $jobTag з тегів?",
-      "ka": "ნამდვილად გსურთ წაშალოთ $jobTag სამუშაოს ტეგებიდან?"
+      "ru": "Вы уверены, что хотите удалить $jobTag из меток?",
+      "en": "Are you sure you want to remove $jobTag from job labels?",
+      "ua": "Ви впевнені, що хочете видалити $ jobTag з міток?",
```
Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/WayPageContent.json:508]:
```suggestion
      "ua": "Ви впевнені, що хочете видалити $jobTag з міток?",
```

File: mw-webapp/src/dictionary/WayPageContent.json [hunk 2]
```diff
@@ -498,27 +498,27 @@
     },
     "deleteFromJobTags": {
       "ru": "Удалить из тегов работ",
-      "en": "Delete from job tags",
-      "ua": "Видалити з тегів робіт",
-      "ka": "წაშალეთ სამუშაოს ტეგებიდან"
+      "en": "Delete from jobs labels",
```
Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/WayPageContent.json:501]: 500 line

Comment (timoxa20) [reply mw-webapp/src/dictionary/WayPageContent.json:501]: Fix 500 line

File: mw-webapp/src/dictionary/WayPageContent.json (hunk 1 above)
Comment (timoxa20) [reply mw-webapp/src/dictionary/WayPageContent.json]: fix 508 line

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**